blueocean-pipeline-api-impl/src/test/java/io/jenkins/blueocean/rest/impl/pipeline/RestartStageTest.java

```diff
@@ -70,7 +73,7 @@ public void restart_stage() throws Exception {
                                       + "/runs/1/nodes/" + res.get( "id" ) + "/restart",
                                   restartMap);
 
-        assertEquals( QueuedBlueRun.class.getName(), restartResult.get( "_class" ) );
+        assertThat(restartResult.get("_class"), anyOf(is(QueuedBlueRun.class.getName()), is(PipelineRunImpl.class.getName())));
```

Comment:
Unrelated to this PR, but most of my PR builds were tripping this assertion saying that the class name was `PipelineRunImpl`. Since this is queuing a build that eventually is scheduled, this test was inherently flaky, and the lines below explicitly deal with the case where the build has been taken off the queue and scheduled, so we do the same here. This made the test suite stable enough for me to eventually get a passing CI run.

Reply:
thanks for fixing this